setup.sh

```diff
@@ -74,11 +75,32 @@ done
   echo "]"
 }
 
+trusted_validator_membership_change_json() {
```

Comment:
This is unnecessary, @d4hines Initially, the file needs to be an empty array json file.